canonical-websites/snapcraft.io
# Measure territories

Depends on https://github.com/canonical-websites/snapcraft.io/pull/197 landing first.

## Summary
- Added territories graph to publisher metrics
- Converted public territories graph to es6 module and added a build step for public js
- Moved snap-metrics build to publisher build, so we now have dist/public.js and dist/publisher.js.

## QA
- Checkout the branch
- ./run serve
- http://0.0.0.0:8004/snaps/dwarf-fortress/measure or http://snapcraft.io-pr-202.run.demo.haus/snaps/dwarf-fortress/measure
- Check all graphs work correctly
- Ensure the map show's "x daily active users" in tooltips
- Go to http://0.0.0.0:8004/dwarf-fortress or http://snapcraft.io-pr-202.run.demo.haus/dwarf-fortress
- Ensure the map still works as before and the map does not show "x daily active users" in tooltips
